doc/adrs/069_add_support_for_children_layout_strategy.adoc

```diff
@@ -0,0 +1,103 @@
+= ADR-069 - Add support for children layout strategy
+
+== Context
+
+We want to improve layout engines (ELK and Incremental) to make them more powerful. We need them to be more powerful to support existing node types and two new types of node, compartment and svg image.
+
+== Decision
+
+We have chosen to separate the behavior that does the layout of a node from the behavior that does the layout of the node children.
+
+The existing node style which is used to handle list node, list item node, rectangular node and image node will be split into three types of nodes:
+- rectangular node style (has an option to consider the label is inside a header),
+- image node style (has an attribute to define the image),
+- icon label node style.
+Each node style will be used to describe how the node is represented in the diagram.
+
+We will introduce the children layout strategy to describe how the layout of node children is done.
+At first, we will support two kind of children layout strategy:
+- free form layout strategy
+- list layout strategy
+- no child layout strategy
+
```

Comment:
Maybe mention that the diagram itself (the implicit parent of top-level nodes) behaves as if it had a "free form" layout strategy.